driver-core/src/test/java/com/datastax/driver/core/ProtocolV1Test.java

```diff
@@ -24,7 +25,7 @@
 /**
  * Tests targeting protocol v1 specifically.
  */
-@CCMConfig(version = "1.2.19")
+@CassandraVersion(major = 1.2, exactRequired = true)
```

Comment:
Hrmm, unfortunately it doesn't seem like this is an option because there isn't a way for me prevent it creating the CCM cluster other than at annotation level, so may need to keep this in here for now.